app/renderer-jsx/app.jsx

```diff
@@ -268,10 +268,31 @@ class AppContainer extends React.Component {
       Enter: this.handleEnter.bind(this),
       Tab: this.handleTab.bind(this)
     };
+    const ctrlKeyHandlers = {
+      P: this.handleSelection.bind(this, -1),
+      p: this.handleSelection.bind(this, -1),
+      K: this.handleSelection.bind(this, -1),
+      k: this.handleSelection.bind(this, -1),
+      N: this.handleSelection.bind(this, 1),
+      n: this.handleSelection.bind(this, 1),
+      J: this.handleSelection.bind(this, 1),
+      j: this.handleSelection.bind(this, 1)
```

Comment:
Actually these shortcuts generally work even if Caps Lock is on. It wouldn't be a bad idea to include the capitalized letters.